mitxpro/urls.py

```diff
@@ -64,7 +64,6 @@
     ),
     path("terms-and-conditions/", index, name="terms-and-conditions"),
     re_path(r"^catalog/", CourseCatalogView.as_view(), name="mitxpro-catalog"),
-    re_path(r"^$", CourseCatalogView.as_view(), name="mitxpro-index"),
```

Comment:
this is also used in `ecommerce/views.py:288`. Please fix it.

Reply:
I'll have to update `ecommerce/views.py` as we are going to let wagtail serve this route rather than hijack it.

Reply:
Fixed